src/sidebar/components/ShareDialog/ExportAnnotations.tsx

```diff
@@ -32,7 +30,11 @@ function ExportAnnotations({
   const exportCount = exportableAnnotations.length;
   const draftCount = store.countDrafts();
 
-  const inputRef = useRef<HTMLInputElement | null>(null);
+  const suggestedGroupName = useMemo(
+    () => suggestedFilename({ groupName: group?.name }),
+    [group],
+  );
+  const [baseFilename, setBaseFilename] = useState<string>();
```

Comment:
```suggestion
  const [customFilename, setCustomFilename] = useState<string>();
```

To me "base" suggests that it is an initial filename to which other elements will be added, as opposed to a custom filename which overrides a default.